tests/test_clipboard.py

```diff
@@ -43,36 +47,30 @@ def start_session(self, pmock, params={}):
 
         return (session, start_session_response.results.get("clipboard_enabled"))
 
-    def test_request_clipboard_and_start_session(self, dbus_test_case):
-        pmock = PortalMock(dbus_test_case, "Clipboard")
-        params = {"force-clipboard-enabled": True}
-        _, clipboard_enabled = self.start_session(pmock, params)
+    def test_request_clipboard_and_start_session(self, portal_mock):
+        _, clipboard_enabled = self.start_session(portal_mock)
 
         assert clipboard_enabled
 
-    def test_clipboard_checks_clipboard_enabled(self, dbus_test_case):
-        pmock = PortalMock(dbus_test_case, "Clipboard")
-        session, clipboard_enabled = self.start_session(pmock)
-        clipboard_interface = pmock.get_dbus_interface()
+    @pytest.mark.parametrize("required_params", ({"RemoteDesktop": {"force-clipboard-enabled": False}},))
```

Comment:
bikeshed: `required_params` sounds like these are, well, required. But these are "just" parameters passed into the test for the test to pass. How about `template_params`?

Reply:
Always happy to take suggestions for names :)